tests/rptest/services/redpanda_cloud.py

```diff
@@ -547,7 +547,7 @@ def _create_cluster_payload(self):
             "resource_group_id": self.current.namespace_uuid,
             "throughput_tier": self.current.product_name,
             "type": _type,
-            "zones": self.current.zones
+            "zones": [self.current.zones]
```

Comment:
Wait I still have an objection here.  If you look at the type hints, `self.current.zones` is `list[str]`.  Can we make THAT true instead of this list construction here?  I.e.

```
self.current.zones = [get_single_zone()]
....
## in JSON payload construction
   "zones": self.current.zones
```

The word is zoneS after all.

Reply:
Updated